### PR TITLE
HPCC-20479 Environment.conf cert file paths

### PR DESCRIPTION
**Pull request by mckellyln**
Signed-off-by: Mark Kelly <mark.kelly@lexisnexisrisk.com>

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [x] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [ ] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Testing:

Started dafilesrv various ways with these settings.

### CODE REVIEW

**Comment by mckellyln** [comment]
@Michael-Gardner please review

File: initfiles/etc/DIR_NAME/environment.conf.in [hunk 1]
```diff
@@ -36,10 +36,10 @@ mpTraceLevel=0
 
 #Specify location of HPCC PKI public/private key files
 # note: if HPCCPassPhrase specified it must be encrypted
-#HPCCPassPhrase=
-#HPCCCertificateFile=/home/hpcc/.ssh/cert.cer
-#HPCCPublicKeyFile=/home/hpcc/.ssh/id_rsa.pub
-#HPCCPrivateKeyFile=/home/hpcc/.ssh/id_rsa
+#HPCCPassPhrase=XC+0Byt45EYD6qV5++8HW+lgsYB/nGud8FtgrubKdLk=
+#HPCCCertificateFile=/home/hpcc/certificate/certificate.pem
```
Comment (Michael-Gardner) [inline initfiles/etc/DIR_NAME/environment.conf.in]: Looks good, but we might want to do something like this so it's not static and will change if we modify options at build time.

#HPCCCertificateFile=${HOME_DIR}/${RUNTIME_USER}/certificate/certificate.pem

And then similar with the other two files.

**Review by Michael-Gardner — APPROVED** [review]

**Comment by richardkchapman** [comment]
@mckellyln

**Comment by mckellyln** [comment]
@Michael-Gardner ok, good idea.  I change the path to be ${HOME_DIR}/${RUNTIME_USER}.
Please re-review.

**Comment by Michael-Gardner** [comment]
@mckellyln Looks good.

**Comment by mckellyln** [comment]
@richardkchapman ok to merge

**Pull request merged**